single-product.html

```diff
@@ -0,0 +1,673 @@
+                            fill="currentColor"></path>
+                    </svg></a>
+
+                <a class="homepage-nav-links" href="./login.html" target="_blank">
+                    <div>LOGIN</div>
+                </a>
+
+                <!-- Avatar -->
+                <a href="#">
+                    <div class="avatar avatar-sm">
+                        <img src="../Images/pingu.jpg" alt="penguin-w-headphones">
+                    </div>
+                </a>
+            </div>
+        </div>
+        <!-- nav bar ends here -->
+
+        <!-- sidebar prod container starts here -->
+        <div class="sidebar-products-container">
+            <div class="sidebar-container">
+                <h1 class="filter-heading">FILTER</h1>
+                <h2 class="sidebar-cmp-heading">CATEGORY</h2>
+                <ul class="ul">
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">xyz</label>
+                            </div>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">xyz</label>
+                            </div>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">xyz</label>
+                            </div>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">xyz</label>
+                            </div>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">xyz
+                                </label>
+                            </div>
+                        </a>
+                    </li>
+                </ul>
+                <ul class="ul">
+                    <h2 class="sidebar-cmp-heading">BRANDS</h2>
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">Brand</label>
```

Comment:
A suggestion: well you have wrapped the same id with multiple labels. so when I click on the labels they actually behaving differently. even if I click on other checkboxes it's only selecting the first one.  Just an Exp: Users can select multiple checkboxes right what you can do is give a separate id for every checkbox and wrap ids with labels by using for attribute.

Reply:
thanks for pointing out I didn't know. Will do the changes asap :)